python-vcstools moved out of incomming for build_common_image.bas

Whilst building the common-image for subt, It seems that python-vcstool has been moved out of incomming. 

I was able to get the install to continue by installing it via pip. 

so replaced the python-vcstool  with python-pip and added a new line in the dockerfile:
RUN pip install vcstool

```
@@ -23,14 +23,14 @@ RUN echo "deb http://packages.ros.org/ros/ubuntu bionic main" > /etc/apt/sources
   && apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
 
 # install ROS (including dependencies for building packages) and other packages
-RUN apt-get -qq update && apt-get -q -y install \
+RUN apt-get  update && apt-get -y install \
   build-essential \
   git \
   python-rosdep \
   python-rosinstall \
+  python-pip \
   python-rosinstall-generator \
   python-wstool \
-  python-vcstool \
   python3-colcon-common-extensions \
   ros-melodic-desktop \
   ros-melodic-teleop-twist-keyboard \
@@ -41,6 +41,7 @@ RUN apt-get -qq update && apt-get -q -y install \
   && rm -rf /var/lib/apt/lists/* \
   && apt-get -qq clean
 
+RUN pip install vcstool
 # sdformat8-sdf conflicts with sdformat-sdf installed from gazebo
 # so we need to workaround this using a force overwrite
 # Do this before installing ign-gazebo
@@ -101,4 +102,4 @@ RUN [ "/bin/bash" , "-c" , \
 RUN mkdir -p ${SOLUTION_WS_PATH}/src
 
 # When running a container start in the developer's home folder
-WORKDIR /home/$USERNAME
\ No newline at end of file
+WORKDIR /home/$USERNAME

```

The build was successful, but with a few error and warning messages:
```
Step 5/23 : RUN echo "deb [trusted=yes] http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list   && wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add -   && apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654   && apt-get -qq update && apt-get -q -y install   ignition-blueprint   ros-melodic-ros-ign   && rm -rf /var/lib/apt/lists/*   && apt-get -qq clean
 ---> Running in 1a049d8e2b61
--2020-06-23 17:23:09--  http://packages.osrfoundation.org/gazebo.key
Resolving packages.osrfoundation.org (packages.osrfoundation.org)... Warning: apt-key output should not be parsed (stdout is not a terminal)
52.52.171.73
Connecting to packages.osrfoundation.org (packages.osrfoundation.org)|52.52.171.73|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1772 (1.7K) [application/octet-stream]
Saving to: 'STDOUT'

     0K .                                                     100%  193M=0s

2020-06-23 17:23:09 (193 MB/s) - written to stdout [1772/1772]

OK
Warning: apt-key output should not be parsed (stdout is not a terminal)
Executing: /tmp/apt-key-gpghome.UGhReobHUz/gpg.1.sh --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
```
```
Step 21/23 : RUN [ "/bin/bash" , "-c" ,   "source /opt/ros/melodic/setup.bash   && sudo apt-get -qq update   && rosdep install --from-paths src --ignore-src -r -y   && colcon build   && sudo rm -rf /var/lib/apt/lists/*   && sudo apt-get -qq clean" ]
 ---> Running in 29aaf31fa931
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
subt_ign: Cannot locate rosdep definition for [ignition-transport7]
subt_ros: Cannot locate rosdep definition for [ignition-transport7]
subt_communication_broker: Cannot locate rosdep definition for [ignition-transport7]
subt_communication_model: Cannot locate rosdep definition for [ignition-common3]
subt_rf_interface: Cannot locate rosdep definition for [ignition-math6]
Reading package lists...
```

```
Finished <<< subt_ign [26.5s]
--- stderr: subt_ign
/home/developer/workspaces/dependencies_ws/src/subt/subt_ign/src/CommsBrokerPlugin.cc: In member function 'void subt::CommsBrokerPlugin::UpdateIfNewBreadcrumbs()':
/home/developer/workspaces/dependencies_ws/src/subt/subt_ign/src/CommsBrokerPlugin.cc:327:33: warning: unused variable 'name' [-Wunused-variable]
     for (const auto& [name, pose] : this->breadcrumbs)
                                 ^
---
Starting >>> subt_example
Finished <<< subt_example [6.19s]
Starting >>> x3_control
Starting >>> x4_control
Finished <<< x4_control [1.55s]
Finished <<< x3_control [1.57s]
--- stderr: darknet_ros
/home/developer/workspaces/dependencies_ws/src/darknet_ros/darknet/src/gemm.c: In function 'time_gpu':
/home/developer/workspaces/dependencies_ws/src/darknet_ros/darknet/src/gemm.c:232:9: warning: 'cudaThreadSynchronize' is deprecated [-Wdeprecated-declarations]
         cudaThreadSynchronize();
         ^~~~~~~~~~~~~~~~~~~~~
In file included from /usr/local/cuda/include/cuda_runtime.h:96:0,
                 from /home/developer/workspaces/dependencies_ws/src/darknet_ros/darknet/include/darknet.h:14,
                 from /home/developer/workspaces/dependencies_ws/src/darknet_ros/darknet/src/utils.h:5,
                 from /home/developer/workspaces/dependencies_ws/src/darknet_ros/darknet/src/gemm.c:2:
/usr/local/cuda/include/cuda_runtime_api.h:957:57: note: declared here
 extern __CUDA_DEPRECATED __host__ cudaError_t CUDARTAPI cudaThreadSynchronize(void);
                                                         ^~~~~~~~~~~~~~~~~~~~~
---
[Processing: darknet_ros]
[Processing: darknet_ros]
[Processing: darknet_ros]
[Processing: darknet_ros]
Finished <<< darknet_ros [2min 44s]

Summary: 40 packages finished [3min 4s]
  2 packages had stderr output: darknet_ros subt_ign
Removing intermediate container 29aaf31fa931
 ---> 8ac02145379d
Step 22/23 : RUN mkdir -p ${SOLUTION_WS_PATH}/src
 ---> Running in ee03b3e32c9a
Removing intermediate container ee03b3e32c9a
 ---> 61721027e533
Step 23/23 : WORKDIR /home/$USERNAME
 ---> Running in 94fff0c7db40
Removing intermediate container 94fff0c7db40
 ---> e8a692e9970a
Successfully built e8a692e9970a
Successfully tagged subt_hello_world_common:latest
```

--- a/docker/hello_world_common/Dockerfile
+++ b/docker/hello_world_common/Dockerfile
@@ -30,7 +30,7 @@ RUN apt-get -qq update && apt-get -q -y install \
   python-rosinstall \
   python-rosinstall-generator \
   python-wstool \
-  python-vcstool \
+  python3-vcstool \
   python3-colcon-common-extensions \
   ros-melodic-desktop \
   ros-melodic-teleop-twist-keyboard \

